core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/NativeImageBuildStep.java

```diff
@@ -225,6 +227,11 @@ public NativeImageBuildItem build(NativeConfig nativeConfig, NativeImageSourceJa
             } else {
                 command.add("-H:-AddAllCharsets");
             }
+            if (nativeConfig.addAllTimeZones) {
+                command.add("-H:+IncludeAllTimeZones");
+            } else {
+                command.add("-H:-IncludeAllTimeZones");
```

Comment:
I wonder if it should be Optional and if we should do nothing if not defined.

Because it could not be defined in the config and then the user could override it with the additional build args.

Reply:
GraalVM has the bad habit of adjusting their flags over time; setting them explicitly has been useful before.

But it's an interesting point that then the can't easily override it. Let's amit the explicit disablemnet until we introduce (later) a configuration option?

Reply:
@machi1990 could you remove the `command.add("-H:-IncludeAllTimeZones");` line to follow @gsmet 's suggestion?

Reply:
Sorry I realize all the typos in my previous comment made it unclear :)

Reply:
Thanks for bringing this one, I had previously forgotten about it. I have pushed a new version without it @gsmet @Sanne